apps/web/pages/apps/installed.tsx

```diff
@@ -81,122 +81,92 @@ function ConnectOrDisconnectIntegrationButton(props: {
   );
 }
 
-function IntegrationsContainer() {
+type AppOutput = inferQueryOutput<"viewer.integrations">["items"][0];
+
+interface IntegrationsContainerProps {
+  variant: App["variant"];
+  className?: string;
+}
+
+const IntegrationsContainer = ({ variant, className = "" }: IntegrationsContainerProps): JSX.Element => {
   const { t } = useLocale();
-  const query = trpc.useQuery(["viewer.integrations"], { suspense: true });
+  const query = trpc.useQuery(["viewer.integrations", { variant }], { suspense: true });
+  const installedFilter = (app: AppOutput) => app.credentialIds.length > 0 || app.isGlobal;
```

Comment:
And we need to make sure we filter out not installed apps, as the Installed Apps page was repurposed to only show that